components/navigation.html

```diff
@@ -0,0 +1,37 @@
+<nav class="navigation">
+  <a class="nav__link" href="#home"
```

Comment:
For correct BEM naming convention:
```suggestion
  <a class="navigation__link" href="#home"
```